internal/backends/android-activity/java/SlintAndroidJavaHelper.java

```diff
@@ -464,6 +465,16 @@ public int color_scheme() {
     public Rect get_view_rect() {
         Rect rect = new Rect();
         mActivity.getWindow().getDecorView().getWindowVisibleDisplayFrame(rect);
+        WindowInsets insets = mActivity.getWindow().getDecorView().getRootView().getRootWindowInsets();
+        if (insets != null) {
+            int dx = rect.left - insets.getSystemWindowInsetLeft();
```

Comment:
The docs suggest that this is already deprecated API:

> This method was deprecated in API level 30.
> Use [getInsets(int)](https://developer.android.com/reference/android/view/WindowInsets#getInsets(int)) with [Type#systemBars()](https://developer.android.com/reference/android/view/WindowInsets.Type#systemBars()) instead.

It also seems that those insets can change (beyond rotation). Maybe it's worth reacting to that (but not in this PR :)

Reply:
yes, but I'd like to try to stay compatible with Android 9 (API level 28)

We do react to that. We should hopefully recieve a MainEvent::ContentRectChanged from android-activity crate.